string anchor_id
float32 radius
float32 variance
